src/Workspaces/Core/Portable/Workspace/Solution/Checksum_Factory.cs

```diff
@@ -14,41 +15,44 @@ namespace Microsoft.CodeAnalysis
     // all these are just helper methods
     internal partial class Checksum
     {
+        private static readonly ObjectPool<IncrementalHash> s_objectPool =
```

Comment:
💡 Consider using a more specific name for this pool field

Reply:
sure.